### PR TITLE
Trip (non)-compact view

### LINKED ISSUE #72: Move grey cards to popovers

### LINKED ISSUE #77: Make frontend 3.0
image

- [ ] Build out regular view
- [ ] Build out compact view
- [ ] Remove this stuff in `viewSlice.js`: setContentFullscreen, setContentNonFullscreen,

### PR DESCRIPTION
**Pull request by Graywing13**
Closes #77 

- Fix blackGradient
- Make teacups use popovers 
- Add `TripSidePanelSingle.js`
- Move `SeeMoreDrawer.js` out of TripViewContent.js
- Adjust scrollbar colors

### CODE REVIEW

File: client/src/components/Views/TripSidePanelSingle.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+export function TripSidePanelSingle() {
+  const activeDayNumber = useSelector((state) => state.view.activeDayNumber)
+  const dispatch = useDispatch()
+  const [dayDetails, setDayDetails] = useState()
+
+  useEffect(() => {
+    // TODO this logic needs to fetch the right trip ID when we have real data
+    setDayDetails(TripViewJson[activeDayNumber - 1])
+  }, [activeDayNumber])
```
Comment (NLmeng) [inline client/src/components/Views/TripSidePanelSingle.js]: should we reset activeDayNumber back to 1 when activeTripId is changed (when a user click on a new trip)

File: client/src/components/Views/SeeMoreDrawer.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <>
+      {renderShowMoreLessButton}
+      <div
+        className={`pointer-events-auto w-full overflow-y-auto bg-transparent 
+          ${isContentFullscreen ? 'h-1/2 p-4 pt-0' : 'h-0'}`}
+      >
+        {seeMoreCardContent}
+      </div>
+    </>
+  )
```
Comment (NLmeng) [inline client/src/components/Views/SeeMoreDrawer.js]: maybe add smooth transitions? currently look kinda choppy when clicking 'See More'. btw using max-h works a lot better. i tried sth like this: 
```
className={`pointer-events-auto w-full overflow-y-auto bg-transparent 
          ${
            isContentFullscreen ? 'max-h-[50vh] p-4 pt-0' : 'max-h-0'
          } transition-all duration-500 ease-in-out`}
```

**Review by NLmeng — COMMENTED** [review]
code looks good. some feedbacks:

File: client/src/components/Views/TripSidePanelSingle.js (hunk 1 above)
Comment (Graywing13) [reply client/src/components/Views/TripSidePanelSingle.js]: oh good idea

File: client/src/components/Views/SeeMoreDrawer.js (hunk 2 above)
Comment (Graywing13) [reply client/src/components/Views/SeeMoreDrawer.js]: oh interesting i wonder why, tyty

File: client/src/components/Views/TripSidePanelSingle.js (hunk 1 above)
Comment (Graywing13) [reply client/src/components/Views/TripSidePanelSingle.js]: added change to reducer

**Review by NLmeng — APPROVED** [review]
lgtm

**Pull request merged**